frontend/src/includes/locale/nl.json

```diff
@@ -800,9 +800,9 @@
                 "required": "Er is minimaal één verwachte antwoordcode vereist."
             },
             "headers": {
-                "placeholder": "Inhouds type: toepassing/json",
+                "placeholder": "Inhoud-Type: toepassing/json",
```

Comment:
this should probably still say `Content-Type: application/json` since it's a defined MIME type and a protected header.

Reply:
Some of the translation are really just placeholders. It was easier to put this data in this file than try to store it in another "non-translation" file.